notebooks/community/feature_store/mobile_gaming_feature_store.ipynb

```diff
@@ -0,0 +1,2274 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        time.sleep(1)
</pre><p>why we need a time.sleep here?</p><p> </p><p>This simulation is a bit confusing, where <code>online_sample</code> is a static dict. I think I see what you are trying to do here, but this way of implementation may not be the best representation.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/296/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='109'/>

Reply:
<p>I see. Wondering if the approach from Marc would be ok. I'm refering to Generate test data to trigger alerting from <a href="https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/model_monitoring/model_monitoring.ipynb" target="_blank">model monitoring notebook</a>. WDYT?</p>

Reply:
@morgandu I think now that we are aligned with the function. I would left as it is.